### ISSUE TITLE
How to inlcude custom modules

### ISSUE THREAD

**Issue by shockbytes** [skulpt/skulpt#1074]
Hello,

I'm trying to use `skulpt` to load a python code formatting library `yapf`and use it from `JavaScript` code. The documentation says that it's possible to just drop the source code of the custom modules into the `lib` folder and build `skulpt` by oneself. 

So far, so great. However, it seems like the documentation is lacking a gist where the custom module is loaded and can be accessed from JavaScript. Do you offer any tutorial how I can call my custom Python code in JavaScript?

**Comment by Dustyposa** [comment]
Do you have run cmd `nom run dist` ?

**Comment by shockbytes** [comment]
Yes, however, it seemed unable to resolve the custom modules and I was unable to import code from these modules.

**Comment by Dustyposa** [comment]
Can you past your library's python code ? I will have a try.

**Comment by shockbytes** [comment]
I am trying to run the code formatting library `yapf` (https://github.com/google/yapf) and I try to access the code formatting api through JavaScript. 

Thanks for your help!

**Comment by Dustyposa** [comment]
Oh，this lib can‘t support in skulpt at yet.Because many depends lib not supported.
You can create some simple lib in `src/lib/`, like `src/lib/my_lib.py`, and write some code
Then run `npm run build`
And you can use the`my_lib` in your code.
```python
# my_lib.py

my_var = "this is a var"
```
Use the lib:
```python
import my_lib

print(my_lib.my_var)
```
